chalk-rust-ir/src/lib.rs

```diff
@@ -130,8 +143,15 @@ pub struct TraitDatumBound<I: Interner> {
 pub struct TraitFlags {
     pub auto: bool,
```

Comment:
An "auto trait" is one that is "automatically implemented" for every struct, so long as no explicit impl is given. Examples are `Send` and `Sync`.